proxy/server/server.go

```diff
@@ -229,6 +231,15 @@ func modifyResponse(res *http.Response) error {
 			})
 		}
 
+		// Insert analytics block before </body>
+      match = injectPositionRegex.FindStringIndex(body)
```

Comment:
I guess the indent here breaks `gofmt`, fixing it may mean that it doesn't need to be removed. May be an issue of tabs vs spaces?

Reply:
will check